### PR TITLE
Fix dashboard rendering if accessibility app is disabled

### LINKED ISSUE #23163: NC20: New dashboard show nearly white screen after update from 19.0.3 to 20.0.0
### Steps to reproduce
1. Updated my v19.0.3 instance to v20.0.0

### Expected behaviour
Dashboard should be loading fine like at the conference instance

### Actual behaviour
New version is loading pretty well. The dashboard is nearly white. The menu bar is white but the item like "Files" is clickable. Account icon (avatar) at top right is visible and menu works.

Bildschirmfoto 2020-10-03 um 21 43 45

Tried to disable/enable the dashboard. 
Tried Safari as well.
Tried to clear browser cache.

### Server configuration

**Operating system:**
Ubuntu 18.04

**Web server:**
Apache

**Database:**
PostgreSQL 10.14

**PHP version:**
7.2

**Nextcloud version:** (see Nextcloud admin page)
20.0.0

**Updated from an older Nextcloud/ownCloud or fresh install:**
Updated from 19.0.3

**Where did you install Nextcloud from:**
Official updater

```
Login as admin user into your Nextcloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here.
```
No errors have been found.

**Are you using external storage, if yes which one:** local/smb/sftp/...
no

**Are you using encryption:** yes/no
no

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
no

### Client configuration
**Browser:**
ff v81

**Operating system:**
Mac OS

#### Web server error log
<details>
<summary>Web server error log</summary>

```
No entries
```
</details>

#### Nextcloud log (data/nextcloud.log)
<details>
<summary>Nextcloud log</summary>

```
No entries
```
</details>

#### Browser log
<details>
<summary>Browser log</summary>

```
No entries
</details>

### PR DESCRIPTION
**Pull request by juliusknorr**
Fixes https://github.com/nextcloud/server/issues/23163

To reproduce disable the accessibility app and try to open the dashboard.

### CODE REVIEW

File: apps/dashboard/src/App.vue [hunk 1]
```diff
@@ -143,7 +143,7 @@ export default {
 			statuses: {},
 			background,
 			version,
-			defaultBackground: window.OCA.Accessibility.theme === 'dark' ? prefixWithBaseUrl('flickr-148302424@N05-36591009215.jpg?v=1') : prefixWithBaseUrl('flickr-paszczak000-8715851521.jpg?v=1'),
+			defaultBackground: window.OCA.Accessibility && window.OCA.Accessibility.theme === 'dark' ? prefixWithBaseUrl('flickr-148302424@N05-36591009215.jpg?v=1') : prefixWithBaseUrl('flickr-paszczak000-8715851521.jpg?v=1'),
```
Comment (skjnldsv) [inline apps/dashboard/src/App.vue:146]: Optional chaining ?

Comment (juliushaertl) [reply apps/dashboard/src/App.vue]: Yeah, stupid habits :grin: Done.

**Review by skjnldsv — APPROVED** [review]

**Review by ChristophWurst — APPROVED** [review]

**Comment by TonyGravagno** [comment]
This PR was merged into master but the issue is still present in 20.0.3 and today's new 20.0.4. : https://github.com/nextcloud/server/issues/23163#issuecomment-749778602

**Pull request merged**